ajax/libs/jquery.tablesorter/package.json

```diff
@@ -17,10 +17,7 @@
     "alphanumeric",
     "natural"
   ],
-  "licenses": [
-    "MIT",
-    "deprecated_GPL-2.0+"
-  ],
+  "licenses": "MIT",
```

Comment:
Licenses are MIT and GPL-2.0 as you can see in https://github.com/mottie/tablesorter/#licensing.
Here you need to update `deprecated_GPL-2.0+` to `GPL-2.0` (SPDX format) instead of removing it.
Therefore, you will also need to update your commit message and PR comment.
You can refer other similar PRs.

Reply:
Don't forget to cc the related issue.